app/src/main/java/com/example/teamyuml/decentworkmobile/views/NoticeList.java

```diff
@@ -44,65 +43,38 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_notice_list);
-        fragmentManager = this.getSupportFragmentManager();
-        panelSpinner = findViewById(R.id.panel_spinner);
-        panelSpinnerAdapter();
 
+        toolbar = findViewById(R.id.toolbar);
```

Comment:
Do you use toolbar anywhere else? If not make it method variable instead of class variable